locale/en/blog/weekly-updates/weekly-update.2016-02-22.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Weekly Update - Feb 22nd, 2016
+author: Minwoo Jung (@jmwsoft)
+date: 2016-02-22T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-02-22
+layout: blog-post.hbs
+---
+
+### Node.js News
+Node v4.3.1 (LTS) is released.
+
```

Comment:
we should likely also mention the v4.4.0-rc that we really need people to test out